src/test/java/org/apache/accumulo/proxy/its/SimpleProxyBase.java

```diff
@@ -2640,9 +2611,7 @@ private ByteBuffer s2bb(String cf) {
   }
 
   private Map<String,String> s2pp(String cf) {
-    Map<String,String> toRet = new TreeMap<>();
-    toRet.put("password", cf);
-    return toRet;
+    return Map.of("password", cf);
```

Comment:
This was a trivial thing I changed. I have no idea what the `s2pp` method names were supposed to be... maybe "string to password property"?